bach/tests/unit/bach/test_series_multi_level.py

```diff
@@ -5,9 +5,8 @@
 from sql_models.util import is_postgres, is_bigquery
 from tests.unit.bach.util import get_fake_df_test_data
 
-pytestmark = pytest.mark.skip_athena_todo()  # TODO: Athena
-
 
+@pytest.mark.db_independent('tested function does not care about engine or dialect')
```

Comment:
This does not add anything for me
```suggestion
@pytest.mark.db_independent
```